frontend/src/customAxios.ts

```diff
@@ -50,9 +50,11 @@ const CustomAxios = ({
 
   instance.interceptors.response.use(
     response => {
-      if (lifecycleToasts?.successToast) {
+      if (lifecycleToasts?.successToast && response.status < 300) {
```

Comment:
some errors do not cause a promise rejection, make sure to not treat those as successes.